{"uid":"abb000039f30cfbb","name":"Add project to workspace test","fullName":"tests.workspaces_test.TestWorkspaces#test_add_project_to_workspace","historyId":"557b6733f5784cd469824aeac0e8ff06","time":{"start":1750211768787,"stop":1750211768787,"duration":0},"description":"Selecting and adding a project to workspace","descriptionHtml":"<p>Selecting and adding a project to workspace</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"data","time":{"start":1750211724599,"stop":1750211724599,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"session_request","time":{"start":1750211724599,"stop":1750211724599,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1750211724599,"stop":1750211724599,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1750211724599,"stop":1750211724599,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Selecting and adding a project to workspace","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1750211771474,"stop":1750211771546,"duration":72},"status":"passed","steps":[],"attachments":[{"uid":"5287367d041bace","name":"HTTP logs of http://checkip.amazonaws.com/","source":"5287367d041bace.txt","type":"text/plain","size":358}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"a4d1981c8b398a08","name":"Cookies","source":"a4d1981c8b398a08.json","type":"application/json","size":4117},{"uid":"a026267ca9469bf3","name":"Session Storage","source":"a026267ca9469bf3.json","type":"application/json","size":359},{"uid":"84752f786c0c7437","name":"Local Storage","source":"84752f786c0c7437.json","type":"application/json","size":548},{"uid":"27e65270d0b5adb6","name":"Full Page Screenshot","source":"27e65270d0b5adb6.png","type":"image/png","size":1363867},{"uid":"58e62b709858a29a","name":"URL","source":"58e62b709858a29a.uri","type":"text/uri-list","size":23},{"uid":"f9289aea1db5881d","name":"Network Logs","source":"f9289aea1db5881d.json","type":"application/json","size":48955}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1750211791707,"stop":1750211791707,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"normal"},{"name":"epic","value":"Workspaces"},{"name":"story","value":"WorkSpaces Creation and Editing Functionality"},{"name":"tag","value":"run(order=5)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"workspaces_test"},{"name":"subSuite","value":"TestWorkspaces"},{"name":"host","value":"fv-az1047-956"},{"name":"thread","value":"2070-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.workspaces_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"861c45d6f54e535","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/687//#testresult/861c45d6f54e535","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750125415400,"stop":1750125415400,"duration":0}},{"uid":"cb306831325bbe3d","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/686//#testresult/cb306831325bbe3d","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750039209516,"stop":1750039209516,"duration":0}},{"uid":"d89b322fea8aa612","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/685//#testresult/d89b322fea8aa612","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1749953167817,"stop":1749953167817,"duration":0}}]},"tags":["run(order=5)"]},"source":"abb000039f30cfbb.json","parameterValues":[]}